### ISSUE TITLE
Database setup in Initial Setup wiki

### ISSUE THREAD

**Issue by dondi** [dondi/GRNsight#1011]
We have documented our database setup instructions but they are not linked to from the Initial Setup wiki page. Let’s do that

**Comment by dondi** [comment]
* @ahmad00m and @Onariaginosa to do some initial content/links
* @Sarronnn and @ntran18 can walk through the documentation and provide feedback on how to optimize for new members

**Comment by Onariaginosa** [comment]
Database setup instructions are linked from the Initial Setup wiki page.

**Comment by Onariaginosa** [comment]
We finally fixed the database issue! It wasn't actually a big in the code, Sequelize was rounding the timestamps from PostgreSQL, causing the timestamp to not work when querying for specific genes. I changed the network generation code to  ignore the milliseconds and that did the trick on both @Onariaginosa and @ntran18 's machines. We did a fresh network database install and it worked

**Comment by kdahlquist** [comment]
As noted in issue #1014, @ntran18 is still working on documentation on how to load the database. She's going to edit the PDF from the databases class and select only the items necessary for GRNsight and add it to the wiki.

**Comment by kdahlquist** [comment]
The timestamps included milliseconds, but we are truncating because we don't need this level of precision.

**Comment by ntran18** [comment]
I added one line in the database setup page to go back to initial setup page and continue on the instructions. I also deleted some slide on the instruction page. 
[postgresql-setup-day.pdf](https://github.com/dondi/GRNsight/files/10917355/postgresql-setup-day.pdf)

**Comment by dondi** [comment]
Given the abridged/distilled content of the PDF and some overlapping bullets in the database setup wiki, let’s see if we can skip the need for the separate PDF, so that the user can just read through things

@ahmad00m suggests separating out the commands that are only run once (i.e., for setup) from the commands that one runs every time (placeholder has been included in the https://github.com/dondi/GRNsight/wiki/Initial-Setup wiki)

**Comment by ntran18** [comment]
I have modified the database setup to remove the PDF and combine overlapping bullets. I also added "Reuse Commands" section at the end of wiki page

**Comment by dondi** [comment]
@nchun2 posted some notes on the database documentation in #1046 —please review with her to see if those changes should be made. After resolution, we can close this issue (we will just open a new one if a future documentation glitch is identified)

**Comment by dondi** [comment]
Based on the latest findings in #1046 @nchun2 will check our current database documentation to see if there are critical/immediate changes needed; otherwise, we can close this and open future issues for documentation

By critical/immediate change, we can limit to:
* Existing errors in the documentation
* Missing steps outside of being distinct in Windows

Additive content (e.g., alternative paths/commands to do when on a different platform) can be a separate issue after #1046 is fully resolved

**Comment by dondi** [comment]
Edits have been made to the wiki
